src/Console/RefreshCommand.php

```diff
@@ -42,8 +43,76 @@ public function fire()
           throw new ConfigException('Please set the "locales" config! See https://github.com/andywer/laravel-js-localization#configuration');
         }
 
-        MessageCachingService::refreshCache();
-        ConfigCachingService::refreshCache();
+        switch (Config::get('js-localization.storage')) {
```

Comment:
You use the new methods `generateMessagesFile` and `generateConfigFile` here to refresh the cache data if `Config::get('js-localization.storage') === 'file'`. But the generated static files are not used for serving data via the HTTP service (and that would probably introduce a performance penalty).

This seems confusing, since the refresh command is for refreshing the service cache.